### DISCUSSION TITLE
Exception while using library on windows OS

### DISCUSSION THREAD

**Discussion by Vatsal-Parikh** [eclipse-milo/milo#960] [General]
**Describe the bug**
While using this library to fetch the data via OPC-UA interface and forwarding to other system by using Rabbit mq I always get the nullpointer exception as attached in screenshot. Interestingly this issue is only occuring on my windows PC and it is working without any problem on MAC OS.
![OPCUA-RabbitMq_Bug](https://user-images.githubusercontent.com/91598781/161784522-f11a979f-e0f2-4c75-968b-9212780f42e5.png)

**Comment by kevinherron** [comment]
The NPE is happening deep into code that isn't part of Milo, you'll need to go ahead and debug it. Find out what is null.
